test/Microsoft.TestPlatform.AcceptanceTests/AppDomainTests.cs

```diff
@@ -20,7 +20,7 @@ public class AppDomainTests : AcceptanceTestBase
 {
     [TestMethod]
     [TestCategory("Windows-Review")]
-    [NetFullTargetFrameworkDataSource]
+    [NetFullTargetFrameworkDataSourceAttribute]
```

Comment:
Why we should spell out `Attribute` on this attribute? TestMethod is also an attribute and does not spell it out.